community_topics_workflow.md

```diff
@@ -0,0 +1,63 @@
+# Ansible Community Topics Workflow
+
+## Overview
+
+**Warning:** Subject to frequent updates. This is a "living document", expect it to change as we progress with the Community Topics workflow over the next few months.
+
+This document describes the Ansible Community Topics workflow (hereinafter `Workflow`) to provide guidance on successful resolving the topics in the asynchronous way.
+
+The Workflow is a set of actions that need to be done successively within the corresponding time frames.
+
+After a topic is created, a Steering Committee member (hereinafter `Committee person`) copies the Workflow into the topic and, along the way, marks the checkboxes as complete to track the progress, see and plan the following actions needed to get the topic resolved within acceptable time.
+
+**Note:** If you have any ideas on how the Workflow can be improved, please create an issue in this repository or pull request against this document.
+
+## Workflow
+
+**Note:** This is a rough scenario and it can vary depending on a topic's complexity and other nuances, for example, when there is a mass agreement up-front.
+
+### Preparation stage
+
+- [ ] A Committee person checks the topic's content, asks the author / other persons to provide additional information if needed (adds the `waiting` label in this case).
+- [ ] Adds the topic to the [Board](https://github.com/orgs/ansible-community/projects/2/views/1).
+
+### Discussion stage
+
+- [ ] If the topic is ready to be discussed, the Committee person:
+
+  - [ ] Adds the `discussion_topic` label.
+  - [ ] Moves the topic to the `In Discussion` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+  - [ ] Open the discussion by adding a comment asking the Community and the Committee to take part in it.
+- [ ] No sync discussion is needed (there are no blockers, complications, confusion, or impasses).
+
+### Voting stage
+
+- [ ] Depending on the topic's complexity, after 1-2 weeks pass since the discussion was open, the Committee person formulates vote options based on the prior discussion and gives participants reasonable amount of time to propose changes to the options (no longer than a week). The person summarizes the options in a comment and also establishes a date when the vote begins if there are no objections about the options / vote date.
+- [ ] In the vote date, the vote starts with the comment of a Committee person which opens the vote and establishes a date when the vote ends ($CURRENT_DATE + no longer than 14 days).
```

Comment:
```suggestion
- [ ] In the vote date, the vote starts with the comment of a Committee person which opens the vote and establishes a date when the vote ends ($CURRENT_DATE + no longer than 21 days; usually it should not exceed 14 days, 21 days should only be used if it is known that a lot of interested persons will likely not have time to vote in a 14 days period).
```

Reply:
sounds good, e.g. in case of Christmas period, massive holidays in summer, etc.